sdk/compute/Azure.ResourceManager.Compute/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Features Added
 
+Bumps the api-version
```

Comment:
Are we going to release this update? If so, maybe we should also specify the version and release date.

Reply:
No release plan yet. I will open another PR to prepare release